kie-server-parent/kie-server-controller/kie-server-controller-openshift/src/test/java/org/kie/server/controller/openshift/storage/ServerTemplateConverterTest.java

```diff
@@ -104,6 +106,19 @@ public boolean isKieServerReady() {
         repo.load(TEST_KIE_SERVER_ID);
     }
 
+    private static void setEnvHostName() {
+        try {
+            Map<String, String> env = System.getenv();
+            Class<?> cl = env.getClass();
+            Field field = cl.getDeclaredField("m");
+            field.setAccessible(true);
+            Map<String, String> writableEnv = (Map<String, String>) field.get(env);
+            writableEnv.put(ENV_HOSTNAME, "TEST_ENV");
```

Comment:
Why do you set this environment variable?

Reply:
because of code [1],if System.getenv(ENV_HOSTNAME) is null, the code line will throw an exception.


[1] https://github.com/kiegroup/droolsjbpm-integration/blob/298b5936b345a29312ca5a275e941889ebe65b8a/kie-server-parent/kie-server-services/kie-server-services-openshift/src/main/java/org/kie/server/services/openshift/impl/storage/cloud/KieServerStateOpenShiftRepository.java#L225

Reply:
IMHO it would be better to set this env variable using surefire plugin.

Reply:
I'm not sure, the system env just use in the test case, surefire plugin will set it in global. 
so Please wait @rhtevan feedback fistly.

Reply:
Agree with @sutaakar , simply adding it to surefire plugin at POM will serve the purpose. One thing is puzzling me is that why this issue is popping up all of sudden. Hopefully, it is not some subtle compatibility issues from f8 kubernetes client API. Perhaps, should check see if there were any need to upgrade the fabric8 k8s client.

Reply:
ok Done